<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1249</width>
    <height>944</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>881</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" rowstretch="3,1">
     <item row="1" column="1">
      <widget class="QLCDNumber" name="lcdNumber2"/>
     </item>
     <item row="1" column="0">
      <widget class="QLCDNumber" name="lcdNumber1"/>
     </item>
     <item row="0" column="0">
      <widget class="QDial" name="dial1">
       <property name="maximum">
        <number>1000</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDial" name="dial2">
       <property name="maximum">
        <number>1832</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1249</width>
     <height>49</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
